src/main/scala/dxWDL/base/Extras.scala

```diff
@@ -328,6 +336,16 @@ object Extras {
         }
     }
 
+    private def checkedParseStringFieldReplaceNull(fields: Map[String, JsValue],
+                                        fieldName: String) : Option[String] = {
+        fields.get(fieldName) match {
+            case None => None
+            case Some(JsString(str)) => Some(str)
+            case Some(JsNull) => Some("")
```

Comment:
Do you actually want to make a distinction between None, and Some(JsNull)?

Reply:
Yes we want to make the distinction between JsNull and None. 

We want the user to provide an empty something or null if there are no inputs so they are aware of the parameter. 

We decided on null but that gets converted back to an empty string (so its easier to work with hehe) - Do you think it makes more sense to just get the user to use an empty string?

Reply:
From a user experience perspective, I think it would be nice to have to only provide the app ID in the extras.json if that's all they need.